src/data/sass-vars.json

```diff
@@ -0,0 +1,19903 @@
+[
+  {
```

Comment:
Thought we added these files to the gitignore?

Reply:
I thought so too, I believe it came through when I pulled in staging